### PR TITLE
Dashboard - unstyled.

### PR DESCRIPTION
**Pull request by felipesere**
There are still plenty of todos on the dashboard:

* [x] group leads should be able to delete a questionnaire
* [x] group leads should be able to do download a CSV of a questionnaire
* [x] group leads should be able to go to the summary of a questionnaire
* [x] users should not see advice requests they have already submitted
* [x] users should be able to distinguish between people who have submitted advice for them those who haven't

### CODE REVIEW

File: lib/advisor/core/questionnaire/creator.ex [hunk 1]
```diff
@@ -25,6 +26,6 @@ defmodule Advisor.Core.Questionnaire.Creator do
   end
 
   defp to_advisory(%{advisor_id: advisor_id, id: id}) do
-    %Advisory{advisor: People.find_by_id(advisor_id), advice_id: id}
+    %Advisory{advisor: People.find_by_id(advisor_id), advice_id: id, id: id}
```
Comment (RabeaGleissner) [inline lib/advisor/core/questionnaire/creator.ex]: Just like the TOOD above says :)

File: test/core/advice_test.exs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  @questions [1, 2, 3]
+
+  test "can figure out if an advice has been answered fully" do
+    christoph = advice_with("Christoph Gockel", @questions)
+
+    answer!(christoph, with: %{"1" => "foo", "2" => "bar", "3" => "batz"})
+    assert Advice.completed?(christoph, length(@questions))
+  end
+
+  test "cann tell if advice has not been answered fully" do
```
Comment (RabeaGleissner) [inline test/core/advice_test.exs]: too many "n"s in the word "can"

File: test/web/controllers/dashboard_page_test.exs [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+    conn
+    |> login_as("Rabea Gleissner")
+    |> get("/dashboard")
+    |> html_response(200)
+    |> still_has_to_give_me_advice("Priya Patil")
+    |> still_has_to_give_me_advice("Sarah Johnston")
+  end
+
+  @tag :skip
```
Comment (RabeaGleissner) [inline test/web/controllers/dashboard_page_test.exs]: did you mean to skip this? The functionality is there now, isn't it?

**Comment by felipesere** [comment]
@RabeaGleissner @fabientownsend I've updated the PR
I managed to remove the advisory record and extracted helper methods into two modules `ThroughTheCore` and `ThroughTheWeb`.
Any remaining feedback?

**Comment by RabeaGleissner** [comment]
Looks good to me! Can't think of any other improvements...

**Pull request merged**